spec/09-lexical-structure.md

```diff
@@ -396,8 +396,8 @@ keyword:: one of
   'enddeclare' 'endfor' 'endforeach' 'endif' 'endswitch' 'endwhile' 'eval' 'exit'
   'extends' 'final' 'finally' 'for' 'foreach' 'function' 'global'
   'goto' 'if' 'implements' 'include' 'include_once' 'instanceof'
-  'insteadof' 'interface' 'isset' 'list' 'namespace' 'new' 'or' 'print' 'private'
-  'protected' 'public' 'require' 'require_once' 'return' 'static' 'switch'
+  'insteadof' 'interface' 'isset' 'iterable' 'list' 'namespace' 'new' 'or' 'print'
```

Comment:
`iterable` is not a keyword, it's a reserved class name (similarly to how the scalar type hints are handled).